tests/src/actionContainers/ActionProxyContainerTests.scala

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package actionContainers
+
+import org.junit.runner.RunWith
+import org.scalatest.Finders
+import org.scalatest.junit.JUnitRunner
+
+import ActionContainer.withContainer
+import spray.json.JsNull
+import spray.json.JsNumber
+import spray.json.JsObject
+import spray.json.JsString
+
+@RunWith(classOf[JUnitRunner])
+class ActionProxyContainerTests extends ActionProxyContainerTestUtils {
+
+    def withPythonContainer(code: ActionContainer => Unit) = withContainer("whisk/dockerskeleton")(code)
+
+    behavior of "whisk/dockerskeleton"
+
+    it should "run sample without init" in {
+        val (out, err) = withPythonContainer { c =>
+            val (runCode, out) = c.run(JsNull)
+            runCode should be(200)
+            out should be(Some(JsObject("msg" -> JsString("Hello from bash script!"))))
```

Comment:
You mean specifically why check the result object instead of the status code itself?
Partly my rationale is that this image will end up on dockerhub and is the built in same.
I'm agreeable to removing line 40 if that's what you mean.